applications/consumer-token-type-i.md

```diff
@@ -0,0 +1,235 @@
+  - Centre of Expertise in Computational Finance, Cooperative Research Centre for Technology-Enabled Capital Markets, University of Technology, Sydney.
+  - Statistics in Finance, [Mathematisches Forschungsinstitut Oberwolfach](https://publications.mfo.de/bitstream/handle/mfo/2821/OWR_2004_02.pdf?sequence=1&isAllowed=y)
+
+- Commercial:
+  - CourseAccess - with John Chionglo (CA), circa 1997. University class scheduling, course grade and materials management web site. When `htc` and `idx` files were a thing. Used by Dr. David Walsh to manage a course of 300+ students served from a (dual!) Pentium-Pro.
+  - Taqtiqa - startup, circa 2009-11. Rode the AWS and devops wave. Negotiated commercial and academic use licenses with NYSE.
+  - [Participant](https://doi.org/10.1111/j.1835-2561.2002.tb00191.x) in the Investment and Financial Services Association (IFSA) debate on expensing employee stock options.
+
+- Technical:
+  - Thesis topics: Generalized AutoRegressive Conditional Heteroskedasticity (GARCH) option pricing and derivation of the asymptotic distribution of the deviance statistic, and related functions (estimators).  Awarded Distinction.
+    - Supervisor: [Prof Ross Maller](https://web.archive.org/web/20230605043356/https://researchers.anu.edu.au/researchers/maller-ra).
+    - Examiners:
+      - [Prof Peter Brockwell](https://web.archive.org/web/20230731032804/https://dl.acm.org/profile/81100472556),
+      - [Prof Charl Chiarella](https://doi.org/10.1016/j.jedc.2018.06.001),
+      - [Prof Jin-Chuan Duan](https://web.archive.org/web/20230131093512/https://d.nuscri.org/profiles/duanjc/).
+  - Market microstructure; [Corporate Finance](https://doi.org/10.1111/j.1835-2561.2002.tb00191.x); Investment Management; [Econometrics](http://dx.doi.org/10.1111/1368-423X.t01-1-00090).
+  - Wrote a distributed computing screen saver for thesis calculations. Deployed across 400+ student laboratory computers; Served from dual Pentium-Pro (Win/NT 3/4).
+  - Rust; Ruby; Julia; Python; MPI/MPICH;
+
+- Open Source:
+  - <HTTP://GITHUB.COM/BEGLEYBROTHERS/IGNITE.CHEF> - Chef Library Cookbook to manage Weaveworks Ingite.
+  - <HTTP://GITHUB.COM/BEGLEYBROTHERS/OCIX> - Cross compile toolchains in Open Container Initiative images.
+  - <HTTP://GITHUB.COM/JLENV> - Robust Julia version and environment management: Bash and Chef.
+
+- W3F:
+  - Grant: [Token-economics survey (2022)](https://github.com/w3f/Grants-Program/blob/master/applications/tokenomics-survey-2022.md)
+  - RFP: [Designing UpChain: a framework for securing Substrate runtime upgrades and Substrate network upgrades](https://github.com/w3f/Grants-Program/pull/1309)
+
+### Team Code Repos
+
+Personal
+
+- <https://github.com/taqtiqa-mark>
+- <https://github.com/bbros-dev>
+- <https://github.com/taqtiqa>
+- <https://github.com/BegleyBrothers>
+
+### Team LinkedIn Profiles (if available)
+
+- <https://www.linkedin.com/in/tatiqa/>
```

Comment:
```suggestion
- <https://www.linkedin.com/in/taqtiqa/>
```

Reply:
Hi @taqtiqa-mark. The URL provided does not exist. Can you fix the link, please?